docs/v3.x/migration-guide/README.md

```diff
@@ -17,6 +17,7 @@ If you were upgrading from the `3.0.0-beta.19.5` to `3.2.0`, here are the follow
 
 ## V3 guides
 
+- [Migration guide from 3.2.3 to 3.2.4](migration-guide-3.2.5-to-3.3.0.md)
```

Comment:
I found mistyping [Migration guide from **3.2.3 to 3.2.4**]
It should be [Migration guide from **3.2.5 to 3.3.0**]

Reply:
Looks good now @iicdii or am I missing something?

Reply:
@derrickmehaffy No it looks fine now to me 😀